doku/blueprints/api/v1/resource.py

```diff
@@ -11,27 +11,52 @@
 def login_check():
     pass
 
+@bp.route("/upload/<int:resource_id>", methods=["PUT"])
+def upload(resource_id: int):
+    resource: Resource = get_or_404(
+        db.session.query(Resource).filter_by(id=resource_id)
+    )
+    schema = ResourceSchema(
+        unknown=EXCLUDE, session=db.session, instance=resource, partial=True
+    )
+
+    data = dict(request.form.copy())
+    if request.json is not None:
+        data.update(request.json)
+
+    if request.files.get("source", None) is not None:
+        file: FileStorage = request.files.get("source")
+        data["source"] = file.read()
```

Comment:
This won’t work like this. For Stylesheets, we read the file and save it’s content in the database in the field _source_.

For resources, we need to save the actual file in the resource folder just like we do when creating the resource.

Reply:
This method has not been used and was now removed in a9aa83b7fa88b329146bb701b7c7568376c83540.